journal/original/14-chapter-11.adoc

```diff
@@ -650,6 +666,9 @@ to petition the council against me.
 Now, when I came from the meeting to the inn where I lodged,
 an officer belonging to the council brought me the following order:--
 
+[.embedded-content-document.letter]
```

Comment:
hmm..., this one is sort of interesting, it's kind of alike a short legal document or a warrant. we don't have a specific type for that, so maybe let's add one, i think it could be useful. Let's use `legal`, so this should be `.legal` instead of `.letter`. I'll update the styleguide.